docs/whats-new.md

```diff
@@ -2,6 +2,61 @@
 
 This page contains highlights of each deck.gl release. Also check our [vis.gl blog](https://medium.com/vis-gl) for news about new releases and features in deck.gl.
 
+## deck.gl v8.5
+
+Release date: TBD
+
+
+<table style="border: 0;" align="center">
+  <tbody>
+    <tr>
+      <td>
+        <img style="max-height:200px" src="https://github.com/visgl/deck.gl-data/blob/master/images/whats-new/text-layer-styling.gif?raw=true" />
+        <p><i>TextLayer styling</i></p>
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+### Default transpilation and bundle size
+
+The NPM distribution has dropped IE 11 support in exchange for an almost 20% reduction in size.
+
+| Entry point | 8.5 Bundle (gzipped) | 8.4 Bundle (gzipped) | Comments |
+| ---  | ---                     | ---                 | ---                 |
+| module (dist/esm)  | 398 KB (115 KB)         | 485 KB (128 KB)     | Transpiled, tree-shaking enabled   |
```

Comment:
It is not clear to me what `module/main` refers to here. This should be simple for the reader to understand.

Reply:
IMO there is nothing both concise and clear we can offer here. At least `module`/`main` are standard package.json fields. The "comments" column provides a little more clarity.